include/libpreopen.h

```diff
@@ -119,5 +139,38 @@ int po_preopen(struct po_map *, const char *path);
  */
 struct po_relpath po_find(struct po_map *map, const char *path,
 	cap_rights_t *rights);
-
+/**
+*Prints and error message when an error occurs
```

Comment:
It would be better if we could print an error message into a string (e.g., using `asprintf(3)`) and then let the caller decide what to do with that information.